.travis.yml

```diff
@@ -1,16 +1,25 @@
 sudo: required
 # Note: travis currently does not support listing more than one language so
-# this cheats and claims to only be cpp.  If they add multiple language
-# support, this should probably get updated to install steps and/or
+# this cheats and claims to only be ruby, which is needed to set an rvm
+# environment that works with CocoaPods (for ObjC tests). If they add multiple
+# language support, this should probably get updated to install steps and/or
 # rvm/gemfile/jdk/etc. entries rather than manually doing the work.
-language: cpp
+language: ruby
+
 os:
   - linux
   - osx
+before_install:
```

Comment:
Instead of a before install, can you just do this in the script for the one that needs it?